src/views/Fetchurl.vue

```diff
@@ -1,76 +1,189 @@
+              Enter G&ZeroWidthSpace;C&ZeroWidthSpace;P storage object URL
+            </label>
+            <textarea
+              id="page-fetch-url-name"
+              v-model="gs_source_url"
+              :disabled="isRequesting"
+              placeholder="gs://ucb-datahub-archived-homedirs/2022-2-summer/data100/.....tar.gz"
+              rows="4"
+              required
+              @focus="clearContents"
+            ></textarea>
           </v-row>
-        </v-container>
-
-        <div class="d-flex justify-end mt-4">
-          <button
-            id="fetch-url-button"
-            :disabled="isCreating"
-            aria-controls="page-reader-alert"
-            class="canvas-button canvas-button-primary"
-            type="submit"
-          >
-            <span v-if="!isCreating">Generate 7-day archive download URL</span>
-            <span v-if="isCreating">
-              <v-progress-circular
-                class="mr-2"
-                color="primary"
-                indeterminate
-              />
-              Creating...
-            </span>
-          </button>
+
+          <v-row no-gutters> 
+            <v-col cols="12">
+              <div class="d-flex justify-end w-100">
+                <v-btn
+                  id="generate-url-btn"
+                  aria-describedby="fetch-url-progress"
+                  :aria-disabled="importButtonDisabled"
+                  class="text-no-wrap my-2"
+                  color="primary"
+                  :disabled="importButtonDisabled"
+                  type="submit"
+                >
+                  <span v-if="!isRequesting">Generate 7-day archive download URL</span>
+                  <span v-if="isRequesting">
+                    <SpinnerWithinButton /> Generating 7-day archive download URL...
+                  </span>
+                </v-btn>
+                <span id="fetch-url-progress" class="sr-only" role="status">
+                  <span v-if="isRequesting">Importing Users</span>
+                </span>
+              </div>
+          </v-col>
+
+          </v-row>
+        </form>
+        
+
+
+        <div v-if="wasCreated" class="d-flex">
+          <div>
+            <label for="response-from-gcp" class="fetch-url-label mb-2 mt-3">
+              Success: the following 7-day download U&ZeroWidthSpace;R&ZeroWidthSpace;L was created.
+            </label>
+            <v-text-field id="response-from-gcp" v-model="responseFromGcp" readonly />
+            <form class="bg-transparent border-0 canvas-form" @submit.prevent="copyText">
+              <v-btn
+                    id="copyText"
+                    class="text-no-wrap my-2"
+                    color="primary"
+                    type="submit"
+              >
+                  <span>Copy this URL to your clipboard</span>
+              </v-btn>
+            </form>
+            </div>
+          
+
+
+
+        </div>
```

Comment:
Your call on front-end code style. I prefer a two-space indentation, enforced by the linter. And one empty-line max.

Reply:
[comment as above re: linter] -- Would love for the linter to fix this for me! I guess the way I used it, it did not catch this? https://github.com/gmerritt/hartsfield-from-ripley/commit/a44aa2ade61bde3158b2e27eafa2ce29d98fc56f

Reply:
ohhh maybe I'm needing ` 'indent': ['error', 2],` in my `.eslintrc.js`!